tests/aclorch_ut.cpp

```diff
@@ -0,0 +1,1024 @@
+        if (consumer->m_toSync.find(key) == consumer->m_toSync.end() || op == DEL_COMMAND) {
+            consumer->m_toSync[key] = entry;
+        }
+        /* If an old task is still there, we combine the old task with new task */
+        else {
+            KeyOpFieldsValuesTuple existing_data = consumer->m_toSync[key];
+
+            auto new_values = kfvFieldsValues(entry);
+            auto existing_values = kfvFieldsValues(existing_data);
+
+            for (auto it : new_values) {
+                string field = fvField(it);
+                string value = fvValue(it);
+
+                auto iu = existing_values.begin();
+                while (iu != existing_values.end()) {
+                    string ofield = fvField(*iu);
+                    if (field == ofield)
+                        iu = existing_values.erase(iu);
+                    else
+                        iu++;
+                }
+                existing_values.push_back(FieldValueTuple(field, value));
+            }
+            consumer->m_toSync[key] = KeyOpFieldsValuesTuple(key, op, existing_values);
+        }
+    }
+    return entries.size();
+}
+
+struct AclTestBase : public ::testing::Test {
+    vector<int32_t*> m_s32list_pool;
+
+    virtual ~AclTestBase()
+    {
+        for (auto p : m_s32list_pool) {
+            free(p);
+        }
+    }
+};
+
+struct AclTest : public AclTestBase {
+
+    struct CreateAclResult {
+        bool ret_val;
+
+        vector<sai_attribute_t> attr_list;
+    };
+
+    shared_ptr<swss::DBConnector> m_config_db;
+
+    AclTest()
+    {
+        m_config_db = make_shared<swss::DBConnector>(CONFIG_DB, swss::DBConnector::DEFAULT_UNIXSOCKET, 0);
+    }
+
+    void SetUp() override
+    {
+        assert(gCrmOrch == nullptr);
+        gCrmOrch = new CrmOrch(m_config_db.get(), CFG_CRM_TABLE_NAME);
+
+        assert(sai_acl_api == nullptr);
+        sai_acl_api = new sai_acl_api_t();
+    }
+
+    void TearDown() override
+    {
+        delete gCrmOrch;
+        gCrmOrch = nullptr;
+
+        delete sai_acl_api;
+        sai_acl_api = nullptr;
+    }
+
+    shared_ptr<CreateAclResult> createAclTable(AclTable& acl)
+    {
+        auto ret = make_shared<CreateAclResult>();
+
+        auto spy = SpyOn<SAI_API_ACL, SAI_OBJECT_TYPE_ACL_TABLE>(&sai_acl_api->create_acl_table);
+        spy->callFake([&](sai_object_id_t* oid, sai_object_id_t, uint32_t attr_count, const sai_attribute_t* attr_list) -> sai_status_t {
+            for (uint32_t i = 0; i < attr_count; ++i) {
+                ret->attr_list.emplace_back(attr_list[i]);
+            }
+            return SAI_STATUS_SUCCESS;
+        });
+
+        ret->ret_val = acl.create();
+        return ret;
+    }
+};
+
+TEST_F(AclTest, Create_L3_Acl_Table)
+{
+    AclTable acltable;
+    acltable.type = ACL_TABLE_L3;
+    auto res = createAclTable(acltable);
+
+    ASSERT_TRUE(res->ret_val == true);
```

Comment:
`ASSERT_TRUE(res->ret_val);`
It's already about true or not.
Otherwise we need to use `ASSERT_EQ()`

Reply:
I will fix it.